content/chat/rooms/moderation/hive-dashboard.textile

```diff
@@ -9,4 +9,45 @@ languages:
   - kotlin
 ---
 
-Hive dashboard documentation to go here.
+"Hive Dashboard":https://hivemoderation.com/dashboard is a powerful all-in-one moderation tool, which allows you to set up rules to combine automated AI moderation with human review.
+
+The Hive Dashboard rule is a rule applied to chat rooms which allows you to use Hive's "moderation dashboard":https://docs.thehive.ai/docs/what-is-the-moderation-dashboard to review messages in your chat room.
+
+h2(#fields). Rule fields
+
+- Hive API Key := The API key for your hive account.
+- Chat Room Filter := A regular expression to match the chat room name.
+
+h2(#hive_setup). Hive setup
+
+To use the Hive Dashboard rule, you first need to set up a Hive account and an application.
+
+Once you have done this you can get an API key from the "dashboard settings":https://dashboard.thehive.ai/app/_/settings/api_keys which you can then use to set up the rule.
+
+When the rule is enabled, it will send all published messages in the room to the dashboard for review.
+Please note that messages get sent to Hive _after_ they have been published, so users will see them in the chat room until they have been deleted by a moderation rule.
```

Comment:
Perhaps worth mentioning that this is near-instantaneous in the case of automated moderation rules